ament_tools/verbs/test_pkg/cli.py

```diff
@@ -21,15 +21,27 @@
 from ament_tools.verbs.build_pkg.cli import get_build_type
 from ament_tools.verbs.build_pkg.cli import handle_build_action
 from ament_tools.verbs.build_pkg.cli import get_context as build_pkg_get_context
+from ament_tools.verbs.build_pkg import prepare_arguments as build_pkg_prepare_arguments
 from ament_tools.verbs.build_pkg.cli import run as build_pkg_run
 
 
+def prepare_arguments(parser, args):
+    parser.add_argument(
+        '-n', '--no-build', dest='build_before_test', default=True, action='store_false',
+        help="assumes package is built, only running the tests"
+    )
+    return build_pkg_prepare_arguments(parser, args)
+
+
 def main(opts):
     opts.build_tests = True
     context = build_pkg_get_context(opts)
-    rc = build_pkg_run(opts, context)
-    if rc:
-        return rc
+    if opts.build_before_test:
+        rc = build_pkg_run(opts, context)
```

Comment:
This function does not only perform the build but also the install step. Since there is also an argument `--skip-install` I would expect `--no-build` to only skip the build.